src/Compilers/CSharp/Test/Emit/CodeGen/CodeGenTupleTest.cs

```diff
@@ -20863,7 +20863,7 @@ static void Main()
             comp.VerifyDiagnostics(
                 // (9,11): error CS1503: Argument 1: cannot convert from '(int, <null>)' to 'int'
                 //         M((1, null));
-                Diagnostic(ErrorCode.ERR_BadArgType, "(1, null)").WithArguments("1", "(int, <null>)", "int").WithLocation(9, 11));
+                Diagnostic(ErrorCode.ERR_BadArgType, "(1, null)").WithArguments("1", $"(int, {MessageID.IDS_NULL.Localize()})", "int").WithLocation(9, 11));
```

Comment:
>Diagnostic(ErrorCode.ERR_BadArgType, "(1, null)").WithArguments("1", $"(int, {MessageID.IDS_NULL.Localize()})", "int").WithLocation(9, 11)); [](start = 16, length = 140)

I believe this failure is caused by a real bug in compiler and changing the test will only hide it. See [https://github.com/dotnet/roslyn/issues/24603](https://github.com/dotnet/roslyn/issues/24603)